src/equals.js

```diff
@@ -4,8 +4,7 @@ var _equals = require('./internal/_equals');
 
 /**
  * Returns `true` if its arguments are equivalent, `false` otherwise.
- * Dispatches to an `equals` method if present. Handles cyclical data
- * structures.
+ * Handles cyclical data structures.
  *
  * Dispatches to the `equals` method of both arguments, if present.
```

Comment:
This might (or quite well might not!) be clearer if it included the word "symmetrically".

    * Dispatches symmetrically to the `equals` methods of both arguments, if present

Either way, it should definitely read "method**s**".